docs/topics/migrations.txt

```diff
@@ -476,6 +476,30 @@ You can pass a second callable to
 want executed when migrating backwards. If this callable is omitted, migrating
 backwards will raise an exception.
 
+.. note::
+
+    When migrating data across models spanning more than one app, ``dependencies``
+    should include the latest migration of each app that is involved, otherwise you
+    will get an error: "LookupError: No installed app with label '``myappname``'".
```

Comment:
Maybe it happens in some cases but not all. I think if we say "otherwise you may" instead of "otherwise you will" that'll be fine.